### ISSUE TITLE
2021.1.3 dependency failure on EL7

### ISSUE THREAD

**Issue by quanterium** [phracek/pycharm-community-edition#62]
Attempting to upgrade from 2021.1.1 to 2021.1.3 (and I think I had the issue with 2021.1.2 as well) fails on CentOS 7, due to failure to resolve a dependency for osgi(com.github.jnr.jffi):

```
$ sudo yum update pycharm-community
Loaded plugins: fastestmirror, langpacks, product-id, search-disabled-repos,
              : subscription-manager

This system is not registered with an entitlement server. You can use subscription-manager to register.

Loading mirror speeds from cached hostfile
 * base: sjc.edge.kernel.org
 * epel: mirror.prgmr.com
 * extras: mirror.keystealth.org
 * updates: mirror.sfo12.us.leaseweb.net
Resolving Dependencies
--> Running transaction check
---> Package pycharm-community.x86_64 0:2021.1.1-3.el7 will be updated
---> Package pycharm-community.x86_64 0:2021.1.3-1.el7 will be an update
--> Processing Dependency: osgi(com.github.jnr.jffi) for package: pycharm-community-2021.1.3-1.el7.x86_64
--> Finished Dependency Resolution
--> Finding unneeded leftover dependencies
Found and removing 0 unneeded dependencies
Error: Package: pycharm-community-2021.1.3-1.el7.x86_64 (phracek-PyCharm)
           Requires: osgi(com.github.jnr.jffi)
 You could try using --skip-broken to work around the problem
 You could try running: rpm -Va --nofiles --nodigest
```

**Comment by xvitaly** [comment]
It look like an rpm autodeps regression.

EPEL 7 package:

```
Requires: /bin/sh /usr/bin/env libc.so.6()(64bit) libc.so.6(GLIBC_2.14)(64bit) libc.so.6(GLIBC_2.2.5)(64bit) libc.so.6(GLIBC_2.3)(64bit) libc.so.6(GLIBC_2.4)(64bit) osgi(com.github.jnr.jffi) osgi(com.sun.jna) rtld(GNU_HASH)
```

EPEL 8 package:

```
Requires: /bin/sh /usr/bin/python3.6 libc.so.6()(64bit) libc.so.6(GLIBC_2.14)(64bit) libc.so.6(GLIBC_2.2.5)(64bit) libc.so.6(GLIBC_2.3)(64bit) libc.so.6(GLIBC_2.4)(64bit) rtld(GNU_HASH)
```

Fedora package:

```
Requires: /usr/bin/python3 /usr/bin/sh libc.so.6()(64bit) libc.so.6(GLIBC_2.14)(64bit) libc.so.6(GLIBC_2.2.5)(64bit) libc.so.6(GLIBC_2.3)(64bit) libc.so.6(GLIBC_2.4)(64bit) rtld(GNU_HASH)
```

**Comment by xvitaly** [comment]
Fixed in 2021.2-2.

**Comment by quanterium** [comment]
Thanks. I can confirm that I was able to install 2021.2-2 from COPR on CentOS 7.
